src/checker.hpp

```diff
@@ -148,8 +148,13 @@ struct AttributeContext {
 
 	String  objc_class;
 	String  objc_name;
-	bool    objc_is_class_method;
+    String  objc_selector;
 	Type *  objc_type;
+    Type *  objc_superclass;
+    Type *  objc_ivar;
+	Entity *objc_context_provider;
+	bool    objc_is_class_method   : 1;
```

Comment:
Only use bit fields when you have enough to minimize padding. This isn't worth it.